UI Theme Update

mwodahl opened the pull request:
# PR Details
This PR updates the dark theme to match the following Figma prototype: https://www.figma.com/proto/QG64ugNXpXMUk2vQ1wgPdZ/CV-Manager-Prototype-January-2025?page-id=2143%3A6061&node-id=2146-7796&viewport=-595%2C-659%2C0.22&t=A1CuORABoO0Qexzk-1&scaling=scale-down&content-scaling=fixed&starting-point-node-id=2146%3A6646

## Description

This PR removes the CdotDark theme and updates the existing dark theme to use the colors found in the Figma CV-Manager prototype.

## How Has This Been Tested?

This was tested locally with docker compose and was used when testing the remainder of the UI updates

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] My changes require new environment variables:
  - [ ] I have updated the docker-compose, K8s YAML, and all dependent deployment configuration files.
- [X] My changes require updates to the documentation:
  - [X] I have updated the documentation accordingly.
- [ ] My changes require updates and/or additions to the unit tests:
  - [ ] I have modified/added tests to cover my changes.
- [X] All existing tests pass.

mwodahl committed:
Update admin tab content height

mwodahl committed:
Remove unused css class

mwodahl committed:
Update admin organization page header display

mwodahl committed:
Update organization delete icon

mwodahl committed:
Update admin organization RSU tab table with additional actions, remove title

mwodahl committed:
Add multiselect css overrides

mwodahl committed:
Update intersection actions, remove table title

mwodahl committed:
Remove bottom margin from form-dropdown

mwodahl committed:
Update intersection tab table field width

mwodahl committed:
Update table actions for organization user tab

mwodahl committed:
Remove className from role dropdown

mwodahl committed:
Remove unused css org user class

mwodahl committed:
Update dropdown border override

mwodahl committed:
Update add form styling

mwodahl committed:
Update styling for user, intersection add forms

mwodahl committed:
Update add RSU form styling

mwodahl committed:
Remove unused import

mwodahl committed:
Update help page background color

mwodahl committed:
Remove icon from contact support menu button

mwodahl committed:
Update testing snapshots

mwodahl committed:
Removing contact support button padding

mwodahl committed:
Update text alignment

mwodahl committed:
Update help header margin

mwodahl committed:
Help page testing snapshot update

mwodahl committed:
Update add notification tab to use dialog, updated form UI

mwodahl committed:
Update edit notification to use dialog, updated form UI

mwodahl committed:
Remove unused import statements

mwodahl committed:
Add required navigation elements to add notification test

mwodahl committed:
Update testing snapshots

mwodahl committed:
Update add intersection form to use material UI select dropdown

mwodahl committed:
Reorder RSU table row actions

mwodahl committed:
Update add RSU form to use react select components

mwodahl committed:
Update add user form to use react MUI select components

mwodahl committed:
Update org edit modal to use react MUI form components

mwodahl committed:
Update form id

mwodahl committed:
Update incorrect controlId values

mwodahl committed:
Update edit intersection to use react MUI form components

mwodahl committed:
Update edit RSU form to use React MUI form components

mwodahl committed:
Update edit user form to use React MUI form components

mwodahl committed:
Update menu css style properties

mwodahl committed:
Remove unused css classes

mwodahl committed:
Remove map unused css classes

mwodahl committed:
Remove background color from toggle button

mwodahl committed:
Update menu background color, add configure rsu accordion

mwodahl committed:
Update table format to match UI prototype

mwodahl committed:
Update search field alignment

mwodahl committed:
Remove GenerateRSUErrorsPDF component

mwodahl committed:
Update menu min width

mwodahl committed:
Update counts display to match UI prototype

mwodahl committed:
Update RSU popup to match UI prototype

mwodahl committed:
Move button position, remove title

mwodahl committed:
Remove header, update button

mwodahl committed:
Update input fields to use MUI components, update buttons

mwodahl committed:
Update snmp walk item background color

mwodahl committed:
Move button position, update button style

mwodahl committed:
Update header style, content

mwodahl committed:
Update testing snapshots

mwodahl committed:
Update table header background color

mwodahl committed:
Update control panel to use transparent background, update accordion buttons/formatting

mwodahl committed:
Add plus/minus buttons, additional components

mwodahl committed:
Rename side-panel, update component to use floating action button

mwodahl committed:
Add visual-settings component after removing from control panel accordion

mwodahl committed:
Update map-legend to use floating action button, update formatting

mwodahl committed:
Update table header color

mwodahl committed:
Update form input layout, remove unnecessary imports

mwodahl committed:
Update form input layout

mwodahl committed:
Remove unused imports

mwodahl committed:
Remove unused getTitle function

mwodahl committed:
Remove getTitle function

mwodahl committed:
remove getTitle

mwodahl committed:
Update snapshots

mwodahl committed:
Update intersection test to just test for snapshot

mwodahl committed:
Update rsu test to just take a snapshot

mwodahl committed:
Update org delete menu snapshot

mwodahl committed:
Update button icons

mwodahl committed:
Add header margin

mwodahl committed:
Fix configure button RSU width

mwodahl committed:
Snapshot update

mwodahl committed:
Update button type, remove unused imports

mwodahl committed:
Remove CDOT dark theme, update dark theme palette with colors from prototype

mwodahl committed:
Remove CDOT dark theme from README, sample.env

mwodahl committed:
Merge branch 'develop' into Update/webapp-header-ui

jacob6838 commented on webapp/src/styles/index.ts:
```diff
@@ -199,11 +129,12 @@ const themeMainLight = createTheme({
 
 // Dark Theme - https://www.realtimecolors.com/?colors=dbe5f5-070e18-8ca9de-282687-6350ce&fonts=Inter-Inter
```
Let's remove this link, this was used to generate the colors for the previous theme and is no longer necessary

mwodahl replied:
Removed

jacob6838 reviewed (commented):
Are we intending to match the font of the figma prototype as well?

jacob6838 reviewed (changes requested):
Overall looks great - just a few questions

jacob6838 commented:
Is the mapbox theme update covered in another task?

jacob6838 committed:
Merge branch 'Update/ui-theme' into Update/webapp-map-ui

rgelmis commented on webapp/src/styles/index.ts:
```diff
@@ -50,76 +50,6 @@ export const testTheme = createTheme({
-  },
-})
-
 // Light Theme - https://www.realtimecolors.com/?colors=0a1424-e7eef8-213e73-7978d9-4431af&fonts=Inter-Inter
```
I would argue we can remove the light theme entirely for now, since only the dark theme has been approved and toggling between both was flagged as a "nice to have" by CDOT

jacob6838 replied:
Right now the webapp can read the light vs dark mode from the browser and use that to switch between - I would be in favor of keeping the light theme, 1 because it's already made and could be preferred by some users and deployers, and 2 because it allows us to check our theme compatibility
I do understand that it is more work to maintain both, but I am in favor of keeping it around

rgelmis replied:
Is there a way for the user to toggle between the two themes?

jacob6838 replied:
We don't have a button right now, changing the computer/browser from dark to light mode is how it works currently. A button would be simple to add though, could put it in the new user settings menu?

jacob6838 replied:
It pulls the themes from the WEBAPP_THEME_LIGHT and WEBAPP_THEME_DARK variables - these can both be set to the same theme if theme-switching is not wanted for a particular deployment

rgelmis replied:
Sounds like this is venturing out of the scope of this PR. Keeping the light theme will cause a waterfall of extra work, including updating the theme to use the correct colors, updating individual elements/components to have correct styling per theme, etc etc. 

Let's focus on dark theme only for now, since that was all that was approved by CDOT. If they get more funding for updates like light theme in the future, it's good to know it'll be relatively simple to get that implemented for them. We can always pull the work on light mode back out from the git history, should CDOT approve the update.

@jacob6838 @mwodahl sound good?

jacob6838 replied:
Is it a reasonable compromise to leave the light theme in, remove it from the defaults, and add a comment saying that is not currently being maintained?

rgelmis replied:
As long as it doesn't affect functionality, sure. It will have to be tested to make sure

mwodahl replied:
I've removed the light theme from the defaults in sample.env and added a few notes that it is currently not being maintained. If additional areas need to be modified, please let me know.

jacob6838 replied:
Looks good to me!

rgelmis reviewed (changes requested):
I agree with Jacob- remove the unneeded links/comments, add the new font we discussed, and this should be good to go! Great work!

mwodahl committed:
Update control panel scrollbar to only scroll content

mwodahl committed:
Update scrollbar position, add display toggle

mwodahl committed:
Add toggle for overlapping elements

mwodahl committed:
Update fontFamily property to include museo-slab

mwodahl committed:
Update css files to use museo-slab font

mwodahl committed:
Update typography to use museo-slab by default

mwodahl committed:
Remove Typography font overrides

mwodahl committed:
Update admin table to use sans-serif font for table content

mwodahl committed:
Remove old cdotDark theme from .env.development

mwodahl committed:
Make dark theme default for both browser modes, add note on light theme

mwodahl committed:
Update testing snapshots with museo-slab font

mwodahl commented:
> Are we intending to match the font of the figma prototype as well?

Yes, this has been updated to match the Figma prototype font.

mwodahl commented:
> Is the mapbox theme update covered in another task?

@rgelmis, do we intend to make any changes to the mapbox theme? I didn't notice any changes to the mapbox theme in the prototype so I didn't modify it at all.

mwodahl committed:
Merge branch 'develop' into Update/ui-theme

jacob6838 commented on webapp/src/styles/index.ts:
```diff
@@ -275,7 +224,6 @@ const themeMainDark = createTheme({
-  cdotDark: themeCdotDark,
 }
 
 export const getCurrentTheme = (isDarkTheme: boolean, defaultLightTheme: string, defaultDarkTheme: string) => {
```
On line 230, let's change the default light theme (let theme = THEMES[defaultLightTheme] ?? THEMES.light) to the dark theme

mwodahl replied:
Updated

jacob6838 replied:
This should be updated a bit more now that we are changing the default. At minimum, we should add the following to the getCurrentTheme else if case:
```ts
theme = THEMES[defaultLightTheme] ?? THEMES.light
```
And to re-write the method in a simpler way, I would propose the following
```ts
export const getCurrentTheme = (isDarkTheme: boolean, lightThemeName: string, darkThemeName: string): Theme => {
  // Warnings to user if theme names are not known
  if (darkThemeName && !THEMES[darkThemeName]) {
    console.warn(`Unknown dark theme name: ${darkThemeName}`)
  }
  if (lightThemeName && !THEMES[lightThemeName]) {
    console.warn(`Unknown light theme name: ${lightThemeName}`)
  }
  if (isDarkTheme) {
    return THEMES[darkThemeName] ?? THEMES.dark
  } else {
    return THEMES[lightThemeName] ?? THEMES.light
  }
}
```

And some proposed unit tests for good measure (I can create a separate PR if you would prefer):
```ts
import { THEMES, getCurrentTheme } from './index'

describe('getCurrentTheme', () => {
  const consoleWarnSpy = jest.spyOn(console, 'warn').mockImplementation(() => {})

  afterEach(() => {
    jest.clearAllMocks()
  })

  it('should return the dark theme when isDarkTheme is true and darkThemeName is valid', () => {
    const theme = getCurrentTheme(true, 'light', 'dark')
    expect(theme).toBe(THEMES.dark)
    expect(consoleWarnSpy).not.toHaveBeenCalled()
  })

  it('should return the light theme when isDarkTheme is false and lightThemeName is valid', () => {
    const theme = getCurrentTheme(false, 'light', 'dark')
    expect(theme).toBe(THEMES.light)
    expect(consoleWarnSpy).not.toHaveBeenCalled()
  })

  it('should return the default dark theme when isDarkTheme is true and darkThemeName is invalid', () => {
    const theme = getCurrentTheme(true, 'light', 'invalidDarkTheme')
    expect(theme).toBe(THEMES.dark)
    expect(consoleWarnSpy).toHaveBeenCalledWith('Unknown dark theme name: invalidDarkTheme')
  })

  it('should return the default light theme when isDarkTheme is false and lightThemeName is invalid', () => {
    const theme = getCurrentTheme(false, 'invalidLightTheme', 'dark')
    expect(theme).toBe(THEMES.light)
    expect(consoleWarnSpy).toHaveBeenCalledWith('Unknown light theme name: invalidLightTheme')
  })

  it('should return the default dark theme when isDarkTheme is true and darkThemeName is undefined', () => {
    const theme = getCurrentTheme(true, 'light', undefined as unknown as string)
    expect(theme).toBe(THEMES.dark)
    expect(consoleWarnSpy).not.toHaveBeenCalled()
  })

  it('should return the default light theme when isDarkTheme is false and lightThemeName is undefined', () => {
    const theme = getCurrentTheme(false, undefined as unknown as string, 'dark')
    expect(theme).toBe(THEMES.light)
    expect(consoleWarnSpy).not.toHaveBeenCalled()
  })

  it('should return the default dark theme when isDarkTheme is true and darkThemeName is null', () => {
    const theme = getCurrentTheme(true, 'light', null as unknown as string)
    expect(theme).toBe(THEMES.dark)
    expect(consoleWarnSpy).not.toHaveBeenCalled()
  })

  it('should return the default light theme when isDarkTheme is false and lightThemeName is null', () => {
    const theme = getCurrentTheme(false, null as unknown as string, 'dark')
    expect(theme).toBe(THEMES.light)
    expect(consoleWarnSpy).not.toHaveBeenCalled()
  })
})

```

mwodahl replied:
This is a great suggestion - and I think it makes sense to add in this PR. I've gone ahead and updated the getCurrentTheme method to use this implementation and added these unit tests to index.test.tsx

mwodahl committed:
Remove unused css class

mwodahl committed:
Update ConfigureRSU height, scrolling

mwodahl committed:
Update map menu height

mwodahl committed:
Update map menu to be expanded by default

mwodahl committed:
Update menu container height

mwodahl committed:
Map testing snapshot update

mwodahl committed:
Merge branch 'develop' into Update/webapp-map-ui

mwodahl committed:
Update default light theme to be dark theme

jacob6838 commented:
Drew just found some visual bugs on the SNMP menu deployed in test:
Un-hovered - difficult to read:
![image](https://github.com/user-attachments/assets/b449935f-40b3-436d-be06-62d93632c5ef)

Hovered - easy to read:
![image](https://github.com/user-attachments/assets/554f4560-7dfa-45c7-9ebe-8e2a9e2dfbf2)


Multiple items overlapping:
![image](https://github.com/user-attachments/assets/27f739e1-2f35-4c4c-b342-23ec73e569a0)

mwodahl committed:
Update dark theme hex values to match figma prototype component library

mwodahl commented:
> Drew just found some visual bugs on the SNMP menu deployed in test: Un-hovered - difficult to read: ![image](https://private-user-images.githubusercontent.com/51723791/423711890-b449935f-40b3-436d-be06-62d93632c5ef.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIzOTMwODQsIm5iZiI6MTc0MjM5Mjc4NCwicGF0aCI6Ii81MTcyMzc5MS80MjM3MTE4OTAtYjQ0OTkzNWYtNDBiMy00MzZkLWJlMDYtNjJkOTM2MzJjNWVmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE5VDEzNTk0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY4ZGRjMzhiZDNlNTM1NTU0NGI1NGQzYTdmZTVlZGEyN2EyMzRhMmNmYWNmMzI0MzQ3ZGFkOWZhMGJkODliNDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.WD93cISefK8lP-4fBkC_8yzwgyb8gTQzj12RuTXHMdY)
> 
> Hovered - easy to read: ![image](https://private-user-images.githubusercontent.com/51723791/423711963-554f4560-7dfa-45c7-9ebe-8e2a9e2dfbf2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIzOTMwODQsIm5iZiI6MTc0MjM5Mjc4NCwicGF0aCI6Ii81MTcyMzc5MS80MjM3MTE5NjMtNTU0ZjQ1NjAtN2RmYS00NWM3LTllYmUtOGUyYTllMmRmYmYyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE5VDEzNTk0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk4ZTlmYzU3MGM0MGRlZjc5MTM5NzY0NTI1NGE3NzYyZDEzM2VlZTc3YWRhYjg1NDZkZDAxMmZjYjkyMTliYWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.wdFsuFRtnWqS5ij0cwXJlcbe4PbUd_Zuqz0sHaCJIOM)
> 
> Multiple items overlapping: ![image](https://private-user-images.githubusercontent.com/51723791/423712024-27f739e1-2f35-4c4c-b342-23ec73e569a0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIzOTMwODQsIm5iZiI6MTc0MjM5Mjc4NCwicGF0aCI6Ii81MTcyMzc5MS80MjM3MTIwMjQtMjdmNzM5ZTEtMmYzNS00YzRjLWIzNDItMjNlYzczZTU2OWEwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE5VDEzNTk0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThlZTU1NDk4ODRlM2IyZTNhOThhMWY1ODkxY2U1MWQ1ZWVhM2ZkYWIzY2Q5YTQ0NjE1Y2ZiMjQzODk2NmMxYzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.01_VVCdqC-y2BPuSjPsgX0F27t6q8iMG7Wh3Ii1f6Do)

I'm unsure how these changes were deployed in test, however, these issues are present when running this branch locally. I can verify that these changes are not present when running #175 alongside the theme update.

rgelmis commented on webapp/src/styles/index.ts:
```diff
@@ -225,48 +160,62 @@ const themeMainDark = createTheme({
     text: {
       primary: '#dbe5f5',
       secondary: '#dbe5f5',
       disabled: '#acacac',
```
```suggestion
      primary: '#FFFFFF',
      secondary: '#c7c7c7',
      disabled: '#acacac',
```

mwodahl replied:
Updated

rgelmis commented on webapp/src/styles/index.ts:
```diff
@@ -225,48 +160,62 @@ const themeMainDark = createTheme({
+      main: '#dbe5f5',
       light: '#535297',
       dark: '#161563',
       contrastText: '#fff',
```
```suggestion
      contrastText: '#FAFAFA',
```

mwodahl replied:
Updated

mwodahl committed:
Remove typography font override

rgelmis commented:
The font for body elements like `<p>` and labels should use Trebuchet MS, not Museo-Slab 
![image](https://github.com/user-attachments/assets/1f0e798d-d2b6-4ea1-8f7a-61c3b08b9efe) vs 
![image](https://github.com/user-attachments/assets/bda7b33d-731c-4fee-8111-a9be14eb795b)

mwodahl committed:
Update theme hex values

mwodahl committed:
Add Trebuchet MS font

mwodahl committed:
Add Trebuchet MS as default font for paragraph, Typography body2 variant

mwodahl commented:
> The font for body elements like `<p>` and labels should use Trebuchet MS, not Museo-Slab ![image](https://private-user-images.githubusercontent.com/111303732/427661719-1f0e798d-d2b6-4ea1-8f7a-61c3b08b9efe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDMwOTMyNjgsIm5iZiI6MTc0MzA5Mjk2OCwicGF0aCI6Ii8xMTEzMDM3MzIvNDI3NjYxNzE5LTFmMGU3OThkLWQyYjYtNGVhMS04ZjdhLTYxYzNiMDhiOWVmZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMzI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDMyN1QxNjI5MjhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jZDhmMWUwYzA3YTUyODA2MWVlN2RjMTFlYjRjZTk3MmEyMTNhOTQwZGUxNDJjODhkZGRlMzU1NTAxY2Q0OGY5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.yjDN-wmLwUWXJ25W6Pjb4QoCnykYMfUJrSgfDHDZfzw) vs ![image](https://private-user-images.githubusercontent.com/111303732/427662727-bda7b33d-731c-4fee-8111-a9be14eb795b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDMwOTMyNjgsIm5iZiI6MTc0MzA5Mjk2OCwicGF0aCI6Ii8xMTEzMDM3MzIvNDI3NjYyNzI3LWJkYTdiMzNkLTczMWMtNGZlZS04MTExLWE5YmUxNGViNzk1Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMzI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDMyN1QxNjI5MjhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01MWFmYWZlNTA3NjlmNDIxOTE5YzQ5NjE5MThhZDhkYzA0OWU4MGQxN2RhMjkwODVhYTY4OGYzYzA0ZjllZmVjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.QASplBQji0E5ea-aN6y_BmcAZSh1RuWuDECp2qyTqPk)

Added the Trebuchet MS font and specified it as the default for the `<p>` element. I also added Trebuchet MS as the default font for the `body2` Typography variant.

mwodahl committed:
Update tab text color, font weight

mwodahl committed:
Remove unnecessary div element

mwodahl committed:
Update unselected HorizontalTabs font weight

mwodahl committed:
Update header layout to use Grid component

mwodahl committed:
Remove unnecessary fontWeight property

mwodahl committed:
Update header padding

mwodahl committed:
add capital-case class

mwodahl committed:
Remove unnecessary height attribute from unsuccessful login logo

jacob6838 commented:
> > Is the mapbox theme update covered in another task?
> 
> @rgelmis, do we intend to make any changes to the mapbox theme? I didn't notice any changes to the mapbox theme in the prototype so I didn't modify it at all.

I just noticed that the map in the prototype was darker, and had slightly different colors. Current dark theme map style:
![image](https://github.com/user-attachments/assets/079d04b7-4fc5-40b2-9683-0dfc30b95150)

Prototype map style:
![image](https://github.com/user-attachments/assets/efbd6291-285d-4c02-815b-608018803fbe)

jacob6838 commented:
> > Drew just found some visual bugs on the SNMP menu deployed in test: Un-hovered - difficult to read: ![image](https://private-user-images.githubusercontent.com/51723791/423711890-b449935f-40b3-436d-be06-62d93632c5ef.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIzOTMwODQsIm5iZiI6MTc0MjM5Mjc4NCwicGF0aCI6Ii81MTcyMzc5MS80MjM3MTE4OTAtYjQ0OTkzNWYtNDBiMy00MzZkLWJlMDYtNjJkOTM2MzJjNWVmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE5VDEzNTk0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY4ZGRjMzhiZDNlNTM1NTU0NGI1NGQzYTdmZTVlZGEyN2EyMzRhMmNmYWNmMzI0MzQ3ZGFkOWZhMGJkODliNDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.WD93cISefK8lP-4fBkC_8yzwgyb8gTQzj12RuTXHMdY)
> > Hovered - easy to read: ![image](https://private-user-images.githubusercontent.com/51723791/423711963-554f4560-7dfa-45c7-9ebe-8e2a9e2dfbf2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIzOTMwODQsIm5iZiI6MTc0MjM5Mjc4NCwicGF0aCI6Ii81MTcyMzc5MS80MjM3MTE5NjMtNTU0ZjQ1NjAtN2RmYS00NWM3LTllYmUtOGUyYTllMmRmYmYyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE5VDEzNTk0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk4ZTlmYzU3MGM0MGRlZjc5MTM5NzY0NTI1NGE3NzYyZDEzM2VlZTc3YWRhYjg1NDZkZDAxMmZjYjkyMTliYWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.wdFsuFRtnWqS5ij0cwXJlcbe4PbUd_Zuqz0sHaCJIOM)
> > Multiple items overlapping: ![image](https://private-user-images.githubusercontent.com/51723791/423712024-27f739e1-2f35-4c4c-b342-23ec73e569a0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIzOTMwODQsIm5iZiI6MTc0MjM5Mjc4NCwicGF0aCI6Ii81MTcyMzc5MS80MjM3MTIwMjQtMjdmNzM5ZTEtMmYzNS00YzRjLWIzNDItMjNlYzczZTU2OWEwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE5VDEzNTk0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThlZTU1NDk4ODRlM2IyZTNhOThhMWY1ODkxY2U1MWQ1ZWVhM2ZkYWIzY2Q5YTQ0NjE1Y2ZiMjQzODk2NmMxYzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.01_VVCdqC-y2BPuSjPsgX0F27t6q8iMG7Wh3Ii1f6Do)
> 
> I'm unsure how these changes were deployed in test, however, these issues are present when running this branch locally. I can verify that these changes are not present when running #175 alongside the theme update.

If that is the case, we should merge #175 (into this branch) before merging this PR into develop

mwodahl committed:
Update getCurrentTheme method after removing cdotDarkTheme, add unit tests for getCurrentTheme method

mwodahl committed:
add backgroundColor to masterdiv

mwodahl committed:
Remove unused classes, update helpHeader to use h2 directly

mwodahl committed:
Remove width attribute from BorderedImage

mwodahl committed:
Update Help page to use Stack and Container MUI elements

mwodahl committed:
Update Help/App testing snapshots

mwodahl committed:
add user-info-btn to Header.css

mwodahl committed:
Remove logo class from Header.css

mwodahl committed:
Update Horizontal Tab styling

rgelmis commented:
@jacob6838 not sure why I can't respond to this https://github.com/CDOT-CV/jpo-cvmanager/pull/172#issuecomment-2767262118 directly. The map in the design was the closest map style we could get to the current cv manager map, so the map does not need to be updated to match that

mwodahl committed:
Merge branch 'Update/ui-theme' into Update/webapp-admin-user-settings-ui

mwodahl committed:
Merge branch 'Update/ui-theme' into Update/webapp-header-ui

mwodahl committed:
Merge branch 'Update/ui-theme' into Update/webapp-help-ui

mwodahl committed:
Merge branch 'Update/ui-theme' into Update/intersection-map-ui

mwodahl committed:
Update failing testing snapshots after merging Update/ui-theme

mwodahl committed:
Add contactWrapper class to ContactSupportMenu.tsx

mwodahl committed:
Fix container and stack ordering in Help.tsx

mwodahl committed:
Remove margin, border from BorderedImage.tsx

mwodahl committed:
Update out-of-date testing snapshots

mwodahl committed:
Fix closing tag order on Help.tsx

mwodahl committed:
Update help snapshot to include closing tag reordering

mwodahl committed:
Update divider color to match prototype

mwodahl committed:
Update user dropdown background color, text capitalization, and text color in Header.tsx

mwodahl committed:
Remove unused classes, add capitalize and trebuchet classes

mwodahl committed:
Merge branch 'Update/ui-theme' into Update/webapp-map-ui

mwodahl committed:
Add capital-case, museo-slab classes to App.css

mwodahl committed:
Update default font to be Trebuchet

mwodahl committed:
Merge branch 'Update/ui-theme' into Update/webapp-map-ui

mwodahl committed:
Add overrides for action, toolbar on material table

mwodahl committed:
Update destination IP / message type input to display in a grid

mwodahl committed:
Set ConfigureRSU width, remove grid spacing elements

mwodahl committed:
Add label to message type input

mwodahl committed:
Update table actions, location hover display, and error color

mwodahl committed:
Add map-control-container class

mwodahl committed:
add map-control classes, set mapboxgl triangle pointer color

mwodahl committed:
Add border-radius

mwodahl committed:
Update menu component css values

mwodahl committed:
Remove unnecessary V2X message viewer logic

mwodahl committed:
Add styled button for floating buttons on map

mwodahl committed:
Add class to update button font

mwodahl committed:
Remove map controls from menu, add floating buttons for rsu status/counts display, add missing labels to inputs

mwodahl committed:
Remove comment

mwodahl committed:
Add museo-slab class to delete forwarding button

mwodahl committed:
Update outdated testing snapshots

mwodahl committed:
Merge branch 'Update/ui-theme' into Update/webapp-help-ui

mwodahl committed:
Add default background color to help page

mwodahl committed:
Merge branch 'Update/ui-theme' into Update/webapp-header-ui

mwodahl committed:
add museo-slab font to header text, logout button, user dropdown

mwodahl committed:
Update outdated testing snapshots

mwodahl committed:
Merge pull request #159 from CDOT-CV/Update/webapp-header-ui

Header UI Updates

payneBrandon reviewed (approved):
lgtm!

payneBrandon committed:
Swapping to profiles for run-intersection option

mwodahl committed:
Merge pull request #188 from CDOT-CV/chore/tasks-update

Swapping to profiles for run-intersection option

mwodahl committed:
Default to RSU online status, remove unused state variables

mwodahl committed:
Merge branch 'Update/ui-theme' into Update/webapp-map-ui

mwodahl committed:
Update layout to use stack, update Destination IP input to have fixed label

mwodahl committed:
Remove unused classes

mwodahl committed:
Update top, left/right position and remove unused class

mwodahl committed:
Update top value

mwodahl committed:
Update headerTabHeight

mwodahl committed:
Add message when no data found to counts table

mwodahl committed:
Update top, right values

mwodahl committed:
Update floating button position value

mwodahl committed:
Add SideBarHeader component for sideBarBlock header

mwodahl committed:
Remove nested accordion components, update accordion elevation

mwodahl committed:
Remove unnecessary class

mwodahl committed:
Remove unnecessary close icon from menu components

mwodahl committed:
Add SideBarHeader, update layout to use stack

mwodahl committed:
Add margin-top to msgfwddiv

mwodahl committed:
Add SideBarHeader to DisplayRsuErrors

mwodahl committed:
Merge pull request #160 from CDOT-CV/Update/webapp-help-ui

Help / Contact Support UI Updates

mwodahl committed:
Add SideBarHeader to multiple config, remove duplicate accordion elements, add divider between accordion panels

mwodahl committed:
Update outdated testing snapshots

mwodahl committed:
Remove width from multiple RSU config Box

mwodahl committed:
Fix SNMPSet menu button layout on multiple RSU config

mwodahl committed:
Merge branch 'Update/webapp-map-ui' of https://github.com/CDOT-CV/jpo-cvmanager into Update/webapp-map-ui

mwodahl committed:
Remove background color from menu-container

payneBrandon commented:
@mwodahl looks like a merge conflict

mwodahl committed:
Fix RSU error report generation

mwodahl committed:
Remove unnecessary useTheme reference

jacob6838 commented:
There is a gap at the bottom of all of the pages:
![image](https://github.com/user-attachments/assets/1634208d-d8d0-4338-a2d6-8365d372b126)

mwodahl committed:
Update headerTabHeight value to reflect header changes

mwodahl committed:
Merge branch 'develop' into Update/ui-theme

mwodahl commented:
> There is a gap at the bottom of all of the pages: ![image](https://private-user-images.githubusercontent.com/51723791/436707903-1634208d-d8d0-4338-a2d6-8365d372b126.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDU0Mzk5OTksIm5iZiI6MTc0NTQzOTY5OSwicGF0aCI6Ii81MTcyMzc5MS80MzY3MDc5MDMtMTYzNDIwOGQtZDhkMC00MzM4LWEyZDYtODM2NWQzNzJiMTI2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDIzVDIwMjEzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNmYWM5Y2ZkMWJiY2YwNGU2YjQ2YTZjZTlmZmU2MGI3MTFiNGVkNjI3NDlkMGMyOGUzOWQzZmFiZDI3ZmQyNTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.JEj0yfrqaa5iOlsxt-XKxdAydizzoQfY9sk7cVIibA0)

This was due to an outdated headerTabHeight value and has been updated

mwodahl committed:
Merge branch 'Update/ui-theme' into Update/webapp-map-ui

mwodahl committed:
Update RSURebootMenu success color

mwodahl committed:
Fix MUI divider color opacity

mwodahl committed:
Remove set width for mapbox popup, update padding values

mwodahl committed:
Add Stack element, set width/height for RSU popup

jacob6838 committed:
Merge branch 'develop' into Update/webapp-map-ui

mwodahl committed:
Remove height from snmpitemdiv

mwodahl committed:
Update RSUFirmwareMenu to use light text variations for success/error messages

mwodahl committed:
Update outdated snmpwalkitem test snapshot

mwodahl committed:
Merge pull request #175 from CDOT-CV/Update/webapp-map-ui

Map Page UI Updates

mwodahl committed:
Merge branch 'Update/ui-theme' into Update/webapp-admin-user-settings-ui

mwodahl committed:
Update AdminIntersectionTab tableActions to use iconProps

mwodahl committed:
Add missing iconProps to admin pages

jacob6838 reviewed (approved):
Changes look great!
I am approving, but want to note that the Admin page is broken in this branch, so I don't want to merge this into develop until we have addressed the issues there

mwodahl committed:
Add additional classes to App.css

mwodahl committed:
Set color on AdminOrganizationDeleteMenu icon

mwodahl committed:
Update VerticalTabs to use start icon, remove indicator on selected

mwodahl committed:
Remove unnecessary classes from Header.css

mwodahl committed:
Update table actions / icons to display correctly with AdminTable updates

mwodahl committed:
Add rowActionIcon for additional row action icon color

mwodahl committed:
Update style overrides to use text secondary color

mwodahl committed:
Update button style, remove mb-3 class, update header on add/edit notification

mwodahl committed:
Update table actions with iconProps

mwodahl committed:
Update action override to display icon components correctly, add additional styling to table

mwodahl committed:
Update table actions

mwodahl committed:
Update accordionDetails padding

mwodahl committed:
Update table actions to return correct default value

mwodahl committed:
Remove mb-3 class, update input labels, fix padding for Add/Edit RSU

mwodahl committed:
Remove form-dropdown classes

mwodahl committed:
Remove form-dropdown class

mwodahl committed:
Remove mb-3 class, add labels to text fields for Add/Edit intersection

mwodahl committed:
Remove mb-3 class, add labels to text fields for Add/Edit organization

mwodahl committed:
Remove mb-3 class, add labels to text fields for Add/Edit user

mwodahl committed:
Add ThemeProvider to AdminOrganizationDeleteMenu test

mwodahl committed:
Remove museo-slab font-family override

mwodahl committed:
Update outdated testing snapshots

mwodahl committed:
Merge branch 'Update/ui-theme' into Update/intersection-map-ui

mwodahl committed:
Update font-size, accordion header styling, and fix Asn.1 Decoding panel typography

mwodahl committed:
Update table headers to use correct font and text-transform

mwodahl committed:
Update decoder-entry-dialog to remove unnecessary space

mwodahl committed:
Add two new custom values for intersection map styling

mwodahl committed:
Fix accordion styling, typography font size, and text capitalization

mwodahl committed:
Update control-panel width, set button hover color

mwodahl committed:
Update map-info accordion styling, typography

mwodahl committed:
Merge branch 'develop' into Update/ui-theme

mwodahl committed:
Update outdated SnmpsetMenu testing snapshot

mwodahl committed:
Fix delete icon font-size

mwodahl committed:
Fix incorrect reference point longitude error message

mwodahl committed:
Update padding on row select, fix display on empty rows

mwodahl committed:
Fix display on add/edit organization

mwodahl committed:
Update outdated Admin/User Settings testing snapshots

payneBrandon committed:
Merge pull request #158 from CDOT-CV/Update/webapp-admin-user-settings-ui

Admin & User Settings UI Updates

mwodahl committed:
Merge branch 'Update/ui-theme' into Update/intersection-map-ui

payneBrandon committed:
Merge pull request #174 from CDOT-CV/Update/intersection-map-ui

Intersection Map UI Update

mwodahl committed:
Add capital-case class to map buttons

mwodahl committed:
Fix Display RSU Status button position
